src/Altinn.Common/Altinn.EFormidlingClient/Altinn.EFormidlingClient/Models/Conversation.cs

```diff
@@ -0,0 +1,171 @@
+using System;
+using System.Collections.Generic;
+
+namespace Altinn.Common.EFormidlingClient.Models
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Conversation"/> class.
+    /// </summary>
+    public class Conversation
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Conversation"/> class.
+        /// </summary>
+        /// <param name="id">ID</param>
+        /// <param name="conversationId">ConversationId</param>
+        /// <param name="messageId">MessageId</param>
+        /// <param name="senderIdentifier">SenderIdentifier</param>
+        /// <param name="receiverIdentifier">ReceiverIdentifier</param>
+        /// <param name="processIdentifier">ProcessIdentifier</param>
+        /// <param name="messageReference">MessageReference</param>
+        /// <param name="messageTitle">MessageTitle</param>
+        /// <param name="serviceCode">ServiceCode</param>
+        /// <param name="serviceEditionCode">ServiceEditionCode</param>
+        /// <param name="lastUpdate">LastUpdate</param>
+        /// <param name="finished">Finished</param>
+        /// <param name="expiry">Expiry</param>
+        /// <param name="direction">Direction</param>
+        /// <param name="serviceIdentifier">ServiceIdentifier</param>
+        /// <param name="messageStatuses">MessageStatuses</param>
+        public Conversation(int id, string conversationId, string messageId, string senderIdentifier, string receiverIdentifier, string processIdentifier, string messageReference, string messageTitle, string serviceCode, string serviceEditionCode, DateTime lastUpdate, bool finished, DateTime expiry, string direction, string serviceIdentifier, List<MessageStatus> messageStatuses)
+        {
```

Comment:
we usually don't have constructors for classes as these.. It the values are simply being assigned to the properties without any manupulation it can be removed.

Reply:
As the class is generated automatically, a comment at the top of the class explaining this will be enough. You can disregard other review comments if this is specified.